<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options - PNGLiter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pngliter.qrc">
    <normaloff>:/images/configure</normaloff>:/images/configure</iconset>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="pngGroupBox">
         <property name="title">
          <string>PNG Options</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="levelTLabel">
            <property name="text">
             <string>Optimization Level:</string>
            </property>
            <property name="buddy">
             <cstring>levelSlider</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="levelSlider">
            <property name="whatsThis">
             <string>The optimization level 0 enables a set of optimization operations that require minimal effort. There will be no changes to image attributes like bit depth or color type, and no recompression of existing IDAT datastreams.
The optimization level 1 enables a single IDAT compression trial. The trial chosen is what OptiPNG thinks itâ€™s probably the most effective.
The optimization levels 2 and higher enable multiple IDAT compression trials; the higher the level, the more trials.
The behavior and the default value of this option may change across different program versions. </string>
            </property>
            <property name="maximum">
             <number>7</number>
            </property>
            <property name="pageStep">
             <number>7</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="levelLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="backupCheckBox">
            <property name="text">
             <string>Back up the files</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="errorRecoveryCheckBox">
            <property name="whatsThis">
             <string>Enable error recovery. This option has no effect on valid input files.

The program will spend a reasonable amount of effort to recover as much data as possible, without increasing the output file size, but the success cannot be generally guaranteed. The program may even increase the file size, e.g., by reconstructing missing critical data. Under this option, integrity shall take precedence over file size.
When this option is not used, the invalid input files are left unprocessed.</string>
            </property>
            <property name="text">
             <string>Error recovery</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="outputGroupBox">
         <property name="title">
          <string>Output</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="outputLabel">
            <property name="text">
             <string>Directory:</string>
            </property>
            <property name="buddy">
             <cstring>directoryEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="directoryEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="advancedLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="advancedLabel">
         <property name="text">
          <string>Advanced options allow you to set specific command-line parameters to the OptiPNG executable.
&lt;br&gt;&lt;br&gt;
These commands are avaiable here:
&lt;br&gt;&lt;a href=&quot;http://optipng.sourceforge.net/manual.html&quot;&gt;http://optipng.sourceforge.net/manual.html&lt;/a&gt;
&lt;br&gt;&lt;br&gt;
%1 will be replaced with the file to be optimized.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>85</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="advancedGroupBox">
         <property name="title">
          <string>Advanced Options</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Arguments:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="argumentsEdit">
            <property name="text">
             <string>-o3 %1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pngliter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>levelLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
